### PR TITLE
closes #15

### LINKED ISSUE #15: Make Product Model Tests

### PR DESCRIPTION
**Pull request by gabrielwr**
closes #15

### CODE REVIEW

File: db/models/product.test.js [hunk 1]
```diff
@@ -0,0 +1,79 @@
...
+    , {Product} = db
+    , {expect} = require('chai')
+
+/* global describe it before afterEach */
+
+describe('The `Product` model', () => {
+  before('Await database sync', () => db.didSync)
+  afterEach('Clear the tables', () => db.truncate({ cascade: true }))
+
+  var product;
```
Comment (JeffKandel) [inline db/models/product.test.js]: Use let here
And remove trailing semi-colons

File: db/models/product.test.js [hunk 2]
```diff
@@ -0,0 +1,79 @@
...
+  beforeEach(function(){
+    product = Product.build({
+      name: 'Mens Bike',
+      category: 'Mountain',
+      price: 1000,
+      images: [1, 2, 3],
```
Comment (JeffKandel) [inline db/models/product.test.js]: Are we adding an image table too?  I thought we were going to have URLs here.

File: db/models/product.test.js [hunk 3]
```diff
@@ -0,0 +1,79 @@
...
+      color: 'Red',
+      size: 'Medium',
+      availability: true,
```
Comment (chloe630) [inline db/models/product.test.js]: We think availability should be quantity. And the quantity would be an integer.

File: db/models/product.test.js [hunk 4]
```diff
@@ -0,0 +1,79 @@
...
+      reviewStars: 4.7,
+      description: 'this bike rocks'
+    });
+  });
+
+  describe('Columns exist', () => {
```
Comment (JeffKandel) [inline db/models/product.test.js]: The columns exist test seems redundant when we are testing the properties below.  Can we take this describe block out.

File: db/models/product.test.js [hunk 5]
```diff
@@ -0,0 +1,79 @@
...
+        expect(savedProduct.color).to.equal('Red');
+        expect(savedProduct.size).to.equal('Medium');
+        expect(savedProduct.availability).to.equal(true);
+        expect(savedProduct.reviewStars).to.equal(4.7);
+        expect(savedProduct.description).to.equal('this bike rocks');
+      })
+    });
+  })
+
+  describe('Validations ', () => {
```
Comment (JeffKandel) [inline db/models/product.test.js:50]: I think there should be a lot more validations for a product.
category is required, price is required and greater than 0, images should be urls, color is required, size is required, availability should be integer greater than or equal to zero, review, and description is required.

**Review by JeffKandel — CHANGES_REQUESTED** [review]
Hey Guys, good work, Nice job with the ES6 format.  There are some suggestions below.  Please make the changes and I'll review!

**Pull request merged**